src/client/user/components/UserLinkTable/StatusBar/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Typography, createStyles, makeStyles } from '@material-ui/core'
+import { Alert, AlertTitle } from '@material-ui/lab'
+import StatusBarCompletedIcon from '../../../widgets/StatusBarCompletedIcon'
+import StatusBarFailedIcon from '../../../widgets/StatusBarFailedIcon'
+import StatusBarInProgressIcon from '../../../widgets/StatusBarInProgressIcon'
+import userActions from '../../../actions'
+import useAppMargins from '../../../../app/components/AppMargins/appMargins'
+import { StatusBarVariant } from '../../../reducers/types'
+import { GoGovReduxState } from '../../../../app/reducers/types'
+
+type StyleProps = {
+  appMargins: number
+}
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      alignItems: 'stretch',
+    },
+    error: {
+      backgroundColor: '#FFEDED',
+      color: theme.palette.text.primary,
+    },
+    info: {
+      backgroundColor: '#FDF3E8',
+      color: theme.palette.text.primary,
+    },
+    success: {
+      backgroundColor: '#EAF9E7',
+      color: theme.palette.text.primary,
+    },
+    content: {
+      paddingTop: theme.spacing(2),
+      paddingBottom: theme.spacing(2),
+      [theme.breakpoints.up('md')]: {
+        paddingRight: (props: StyleProps) => props.appMargins,
+        paddingLeft: (props: StyleProps) => props.appMargins,
+      },
+      [theme.breakpoints.down('sm')]: {},
+    },
+    message: {},
+    messageBody: {
+      color: '#767676',
+    },
+    closeButton: {
+      marginRight: theme.spacing(2.5),
+    },
+    iconWrapper: {
+      width: '16px',
+      height: '16px',
+    },
+  }),
+)
+
+const StatusBar = () => {
+  const appMargins = useAppMargins()
+  const classes = useStyles({ appMargins })
+
+  const header = useSelector(
+    (state: GoGovReduxState) => state.user.statusBarMessage.header,
+  )
+  const body = useSelector(
+    (state: GoGovReduxState) => state.user.statusBarMessage.body,
+  )
+  const variant = useSelector(
+    (state: GoGovReduxState) => state.user.statusBarMessage.variant,
+  )
+
+  const hasStatusBarAlert = !!(header || body)
+  const dispatch = useDispatch()
+
+  let colorClass = ''
+  let icon: JSX.Element = <></>
+  switch (variant) {
+    case StatusBarVariant.Error:
+      colorClass = classes.error
+      icon = <StatusBarFailedIcon />
+      break
+    case StatusBarVariant.Info:
+      colorClass = classes.info
+      icon = <StatusBarInProgressIcon />
+      break
+    case StatusBarVariant.Success:
+      colorClass = classes.success
+      icon = <StatusBarCompletedIcon />
+      break
+    default:
+      break
+  }
+
+  const dispatchCloseStatusBar = () => {
+    dispatch(userActions.closeStatusBar())
+  }
+
+  return (
+    <>
+      {hasStatusBarAlert && (
+        <Alert
+          icon={icon}
+          className={`${colorClass} ${classes.root} ${classes.content}`}
```

Comment:
nit: maybe we could merge `classes.root` and `classes.content`, since they only seem to be used together?

Reply:
good idea [b7559d9](https://github.com/opengovsg/GoGovSG/pull/2040/commits/b7559d91c8663c78ed7333f72f265aefd956d326)